cmake/BuildLibElf.cmake

```diff
@@ -0,0 +1,15 @@
+include(ExternalProject)
+
+ExternalProject_Add(libelf
+  PREFIX "vendor/elfutils"
+  URL http://archive.ubuntu.com/ubuntu/pool/main/e/elfutils/elfutils_0.163.orig.tar.bz2
```

Comment:
imo that's ugly too. imo libelf.so is not a big deal, since it's only a dependency in libbcc.so.
The fully static bcc-lua should be statically linking both libbcc.a and libelf.a
and in the future that will be the main tracing tool that won't have any deps.